subprojects/plugin-development/src/main/java/org/gradle/plugin/devel/tasks/ValidatePlugins.java

```diff
@@ -236,26 +223,29 @@ public ConfigurableFileCollection getClasspath() {
     }
 
     /**
-     * Returns whether the build should break when the verifications performed by this task detects a warning.
+     * Specifies whether the build should break when plugin verifications fails.
+     *
+     * @return {@code false} when the build should break on failure, {@code true} when failures should be ignored.
```

Comment:
The `@return` clause is not correct any more.

Reply:
Removed.